.github/workflows/ci.yml

```
@@ -29,72 +28,65 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        platform: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: ["3.8", "3.9", "3.10"]
-        backend: [pyqt5]
-        include:
-          - platform: ubuntu-latest
-            python-version: 3.7
-            backend: pyqt5
+        platform: [ubuntu-latest, macos-13, macos-latest, windows-latest]
+        python-version: ["3.9", "3.10"]
+        backend: [PyQt5]
 
     steps:
       - uses: actions/checkout@v4
-      - uses: conda-incubator/setup-miniconda@v2
+      - uses: conda-incubator/setup-miniconda@v3
         with:
-          miniconda-version: "latest"
-          channels: conda-forge
-          channel-priority: strict
+          miniforge-version: "latest"
           python-version: ${{ matrix.python-version }}
 
-      - uses: tlambert03/setup-qt-libs@v1
-
-      - name: Install Windows OpenGL
-        if: runner.os == 'Windows'
-        run: |
-          git clone --depth 1 https://github.com/pyvista/gl-ci-helpers.git
-          powershell gl-ci-helpers/appveyor/install_opengl.ps1
+      - uses: pyvista/setup-headless-display-action@v3
+        with:
+          qt: true
 
       - name: Install dependencies
         run: |
+          mamba install omero-py
```

```suggestion
          conda install omero-py
```
Looks like mamba issue on windows, lets use conda, should be the same now with modern conda.
If setup is an issue we can switch to micromamba action in the future.